### DISCUSSION TITLE
Welcome to Galadriel-map Discussions!

### DISCUSSION THREAD

**Discussion by VladimirKalachikhin** [VladimirKalachikhin/Galadriel-map#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CzarofAK** [comment]
First of all, thanks Vladimir for your great work!

I am Gabriel an coming from Switzerland. The Galadriel map shall be used in my motorhome. It will be used for:
- Navigation
- Route recording
- AIS display (cypho-150 installed)
I would like to install it on the same Raspberry Pi 3+ as my Victron Venus is running. If possible, i would also like to install the ModeSdeco2 on the same Raspberry - i have a ModeSbeast installed in the camper.

**Reply by VladimirKalachikhin** [reply]
Thanks for the good words. 

The idea with  ADS-B receiver is interesting but unexpectedly. May be.
As regarding the Victron panel -- this has an own web-based control app. Hardly anything else is needed, is not it?

**Reply by CzarofAK** [reply]
I have to rely on my software-friends, as i am only a electronics.
The Galadriel-Map and the Victron Venus should be alright.
the difficult (maybe not for a programmer) part is to install the ModeSdeco2 on the Victron Venus.
Let you know if it worked

**Reply by CzarofAK** [reply]
May i ask something?
i have found out, that the Victron OS is runing a hiawatha server as default.
is it possible to add a hiawatha config into your "emergencykit"?
https://community.victronenergy.com/questions/55309/webserver-on-raspberry-pi-running-venus-os.html

**Reply by VladimirKalachikhin** [reply]
Heh, I've never heard of Victron OS before.
Ok, hiawatha is a web server. It can the PHP-FPM, as Wikipedia says. But what about PHP & PHP-FPM on the Victron OS? As I understand it, Victron OS use SignalK for hardware service, so it javascript. So you must install PHP to Victron OS. Is it possible?
Also -- GaladrielMap need the gpsd for hardware service. I think it's a bad idea to run gpsd with SignalK server in parallel. Easier to rewrite GaladrielMap to javascript and SignalK.

But the idea is interesting...

UPD: If teach GaladrielMap to gets info from SignalK instead gpsd...

**Reply by CzarofAK** [reply]
It seems that Victron has cut down the system in half, or even less - there is not much of the original raspbian left.
There are different versions of the Victron Venus OS
one that runs Signal-K or Red-node, the standard one (the one i have installed) and a version on Raspbian Jessie with packages instead of a full OS.
i guess i am going to try this week to run in to the Jessie with Venus packages. that should be the easiest.

**Reply by VladimirKalachikhin** [reply]
> If teach GaladrielMap to gets info from SignalK instead gpsd...

https://github.com/SignalK/signalk-to-nmea0183

It seems that enough configure gpsd for connecting to port 10110 on Victron for access to all data...
In the case of GaladrielMap on a router, and a GNSS receiver connected to Victron.

But the SIM7600 is very cool.

**Comment by mgrouch** [comment]
Check out this one

https://bareboat-necessities.github.io/my-bareboat/bareboat-os.html

Might be what you looking for

Thanks

**Reply by CzarofAK** [reply]
Thanks, that look also pretty good, but vladimir's its really cool, as it is slim and provides exactly what i like and need.
I try now to install it on a OpenWrt Router (Rpi3B+ with Simcom SIM7600E-H)
